src/main/resources/application.yaml

```diff
@@ -92,10 +92,14 @@ idam:
     microservice: sptribs_case_api
   api:
     url: ${IDAM_API_BASEURL:http://localhost:5000}
+  client:
+    id: ${IDAM_CLIENT_ID:sptribs-case-api}
+    redirect_uri: ${IDAM_CLIENT_REDIRECT_URI:http://localhost:3001/receiver}
+    secret: ${IDAM_CLIENT_SECRET:thUphEveC2Ekuqedaneh4jEcRuba4t2t}
 
   systemupdate:
-    username: ${IDAM_SYSTEM_UPDATE_USERNAME:dummysystemupdate@test.com}
-    password: ${IDAM_SYSTEM_UPDATE_PASSWORD:dummy}
+    username: ${IDAM_SYSTEM_UPDATE_USERNAME:st-test1@mailinator.com}
```

Comment:
Do wen want to revert this username and password?

Reply:
the dummy entries are garbage, suggest we use the Secrets?